packages/firestore/test/unit/specs/existence_filter_spec.test.ts

```diff
@@ -82,6 +108,34 @@ describeSpec('Existence Filters:', [], () => {
     );
   });
 
+  specTest("Existence filter doesn't raise unexpected events", [], () => {
+    const query = Query.atPath(path('collection'));
+    const doc1 = doc('collection/1', 1000, { v: 1 });
+    const doc2 = doc('collection/2', 2000, { v: 2 });
+    return (
+      spec()
+        .userListens(query)
+        .watchAcksFull(query, 1000, doc1)
+        .expectEvents(query, { added: [doc1] })
+        .watchSends({ affects: [query] }, doc2)
+        // Send an existence filter, but don't send a new global snapshot. We
+        // should not see an event.
```

Comment:
I'm not sure what this is testing...  The comment implies that it's not raising an event because we haven't gotten a global snapshot, but as your code exists right now it's not raising an event because there's no mismatch...

If there was a mismatch then it'd raise an event without a global snapshot (which I assert is a bug).